packages/flutter_tools/lib/src/test/flutter_web_platform.dart

```diff
@@ -167,6 +169,41 @@ class FlutterWebPlatform extends PlatformPlugin {
     }
   }
 
+  Future<shelf.Response> _goldenFileHandler(shelf.Request request) async {
+    if (request.url.path.contains('flutter_goldens')) {
+      final Map<String, Object> body = json.decode(await request.readAsString());
+      final String goldenKey = body['key'];
+      // TODO(jonahwilliams): figure out how to correctly translate the
```

Comment:
This is the interesting part, we can clip the screenshot to match the repaint boundary we're testing against